discovery-infra/consts.py

```diff
@@ -37,9 +37,13 @@ class NodesStatus:
     ERROR = "error"
     PENDING_FOR_INPUT = "pending-for-input"
     DAY2_INSTALLED = "added-to-existing-cluster"
+    RESETING_PENDING_USER_ACTION = "resetting-pending-user-action"
 
 
 class ClusterStatus:
+    INSUFFICIENT = "insufficient"
```

Comment:
maybe we can take from assisted-service python client and remove those consts?

Reply:
from what i see in the generated client there is no way get the list of cluster/host states

Reply:
swagger python client isn't generating them nicely :( 
```python
    @status.setter
    def status(self, status):
        """Sets the status of this Cluster.

        Status of the OpenShift cluster.  # noqa: E501

        :param status: The status of this Cluster.  # noqa: E501
        :type: str
        """
        if status is None:
            raise ValueError("Invalid value for `status`, must not be `None`")  # noqa: E501
        allowed_values = ["insufficient", "ready", "error", "preparing-for-installation", "pending-for-input", "installing", "finalizing", "installed"]  # noqa: E501
        if status not in allowed_values:
            raise ValueError(
                "Invalid value for `status` ({0}), must be one of {1}"  # noqa: E501
                .format(status, allowed_values)
            )

        self._status = status
```